CHANGELOG

```diff
@@ -1,6 +1,11 @@
 v3.7.9 (XXXX-XX-XX)
 -------------------
 
+* Follower primaries respond with
```

Comment:
this should go into the 3.7.10 section of the CHANGELOG. 3.7.9 is already released.

Reply:
Done